extension/chrome/elements/composer/composer-input.ts

```diff
@@ -187,6 +190,15 @@ export class ComposerInput extends ComposerComponent {
     return result;
   }
 
+  // We need this method to test imagees in drafts because we can't paste them dirctly in tests.
```

Comment:
A typo above.

It's workable as a workaround but I'd hope we could find some other way to achieve this. If not with puppeteer, then using OS level utilities.